cmake/Config.cmake.in

```diff
@@ -0,0 +1,24 @@
+include("${CMAKE_CURRENT_LIST_DIR}/@WT_TARGETS_EXPORT_NAME@.cmake")
+
+include(CMakeFindDependencyMacro) 
+
+IF(MSVC)
+  #need to turn off auto linking for boost
+  set_property(TARGET WT::wt PROPERTY INTERFACE_COMPILE_DEFINITIONS "BOOST_ALL_NO_LIB=1")
```

Comment:
This is something that should be inherited from Boost, you don't have to set it explictily.